com/AlterETM.py

```diff
@@ -244,11 +231,8 @@ def apply_change(cnn, station, tpar, soln):
     # check if solution exists for this station
     try:
         epar = cnn.get('etm_params', ppar, list(ppar.keys()))
-
         print(' >> Found a set of matching parameters for station ' + station_soln)
```

Comment:
break this line into two? Python let's you do in parenthesis implicitly like this:

```python
 print(' >> Found a set of matching parameters for station '
       + station_soln)
```